components/org.wso2.carbon.identity.api.user.common/src/main/java/org/wso2/carbon/identity/api/user/common/Constants.java

```diff
@@ -57,7 +57,10 @@ public enum ErrorMessage {
                 "The provided userId is invalid."),
         ERROR_CODE_SERVER_ERROR(USER_MANAGEMENT_PREFIX.getPrefix() + "15001",
                 "Unable to retrieve User.",
-                "Server Encountered an error while retrieving the user.");
+                "Server Encountered an error while retrieving the user."),
+        INVALID_TENANT_DOMAIN(USER_MANAGEMENT_PREFIX.getPrefix() + "10002",
+                "Invalid tenant domain.",
+                "Server Encountered an error while retrieving tenantId for tenantDomain.");
```

Comment:
Can we add more context to this? Add the tenant domain at the end? WDYT?

Reply:
implemented